### DISCUSSION TITLE
[Bug] 中文社区的 RSS 订阅失效

### DISCUSSION THREAD

**Discussion by AntoineChn** [zotero-chinese/wiki#209] [求助 / Q&A] [answered]
Zotero 中文社区的管理员，你好，

如题，我测试了下面的两个订阅链接，它们都失效了

- [https://zotero-chinese.github.io/rss.rss](https://zotero-chinese.github.io/rss.rss)
- [https://zotero-chinese.com/rss.rss](https://zotero-chinese.com/rss.rss)

请问，是否更换了新的订阅链接呢？

**Comment by northword** [comment]
网站架构升级，rss 功能尚未恢复，请关注 https://github.com/zotero-chinese/website/pull/42 以追踪进度。

**Comment by northword** [comment, marked as answer]
RSS 已上线，地址为：`https://zotero-chinese.com/feed.xml` 。

`https://zotero-chinese.com/rss.rss` 也可继续使用。

该地址仅包含所有文档，日期为最后更新日期，不包含插件商店、CSL 商店和转换器商店的更新。
